package.json

```diff
@@ -43,7 +42,6 @@
     "karma-junit-reporter": "1.2.0",
```

Comment:
Do we use karma-junit-reporter anywhere?

Reply:
Jenkins

Reply:
Run ‘yarn why’. Is it a peer dependency?